kitsune/sumo/form_fields.py

```diff
@@ -68,17 +68,13 @@ def to_python(self, value):
             else:
                 return []
 
-        users = []
-        usernames = [name.strip() for name in value.split(",") if name]
+        usernames = [name.strip() for name in value.split(",") if name.strip()]
```

Comment:
Why are you changing this? I don't see how the check on the strip() call will help.

Reply:
When I test this in shell, if I pass something like `"Smith, smith, ,, smith1"` as the value, that comprehension will return a username of `" "` that then has to be looked up. I don't think it's vital either way, but why wouldn't I catch it here instead of letting through?